enos/enos-scenario-replication.hcl

```diff
@@ -0,0 +1,682 @@
+      vault_install_dir = local.vault_install_dir
+    }
+  }
+
+  step "add_primary_cluster_nodes" {
+    module = module.vault_cluster
+    depends_on = [
+      step.create_vpc,
+      step.create_primary_backend_cluster,
+      step.create_vault_primary_cluster,
+      step.verify_replicated_data
+    ]
+
+    providers = {
+      enos = local.enos_provider[matrix.distro]
+    }
+
+    variables {
+      ami_id             = step.create_vpc.ami_ids[matrix.distro][matrix.arch]
+      common_tags        = local.tags
+      consul_cluster_tag = step.create_primary_backend_cluster.consul_cluster_tag
+      consul_release = matrix.primary_backend == "consul" ? {
+        edition = var.backend_edition
+        version = matrix.consul_version
+      } : null
+      dependencies_to_install   = local.dependencies_to_install
+      instance_type             = local.vault_instance_type
+      kms_key_arn               = step.create_vpc.kms_key_arn
+      storage_backend           = matrix.primary_backend
+      unseal_method             = matrix.primary_seal
+      vault_cluster_tag         = step.create_vault_primary_cluster.vault_cluster_tag
+      vault_init                = false
+      vault_license             = step.read_license.license
+      vault_local_artifact_path = local.bundle_path
+      vault_install_dir         = local.vault_install_dir
+      vault_artifactory_release = matrix.artifact_source == "artifactory" ? step.build_vault.vault_artifactory_release : null
+      vault_environment = {
+        VAULT_LOG_LEVEL = "debug"
+      }
+      vault_node_prefix         = "newprimary_node"
+      vault_root_token          = step.create_vault_primary_cluster.vault_root_token
+      vault_unseal_when_no_init = matrix.primary_seal == "shamir"
+      vault_unseal_keys         = matrix.primary_seal == "shamir" ? step.create_vault_primary_cluster.vault_unseal_keys_hex : null
+      vpc_id                    = step.create_vpc.vpc_id
+    }
+  }
+
+  step "verify_add_node_unsealed" {
+    module     = module.vault_verify_unsealed
+    depends_on = [step.add_primary_cluster_nodes]
+
+    providers = {
+      enos = local.enos_provider[matrix.distro]
+    }
+
+    variables {
+      vault_instances   = step.add_primary_cluster_nodes.vault_instances
+      vault_install_dir = local.vault_install_dir
+    }
+  }
+
+  step "verify_raft_auto_join_voter" {
+    skip_step = matrix.primary_backend != "raft"
+    module    = module.vault_verify_raft_auto_join_voter
+    depends_on = [
+      step.add_primary_cluster_nodes,
+      step.create_vault_primary_cluster,
+      step.verify_add_node_unsealed
+    ]
+
+    providers = {
+      enos = local.enos_provider[matrix.distro]
+    }
+
+    variables {
+      vault_instances   = step.add_primary_cluster_nodes.vault_instances
+      vault_install_dir = local.vault_install_dir
+      vault_root_token  = step.create_vault_primary_cluster.vault_root_token
+    }
+  }
+
+  step "remove_primary_follower_1" {
```

Comment:
is the `_1` just because its a single follower?

Reply:
It is `follower_1`.  The `vault_get_cluster_ips` module identifies the nodes as `leader`, `follower_1` and `follower_2` so it facilitates removing the identified node in these steps